pyrevolve/genotype/neat_brain_genome/neat_brain_genome.py

```diff
@@ -188,19 +191,19 @@ def clone(self):
 
     @property
     def id(self):
-        return f"{self._neat_genome.GedID()}"
+        return f"{self._neat_genome.GetID()}"  #Change
```

Comment:
change?
GedID :man_facepalming: for myself...

Reply:
#Change: So that I dont forget where I edited the original code in case I have to go back